### ISSUE TITLE
7316-4439 and 7316-7511 RNA-Seq do not contain `KIAA1549::BRAF` fusion

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/OpenPBTA-analysis#441]
#### What data file(s) does this issue pertain to?
histologies, fusion files

#### What release are you using?

v12

#### Put your question or report your issue here.
```r
> v12 %>%
+   filter(cohort_participant_id == "C1100727") %>%
+   select(Kids_First_Biospecimen_ID, sample_id, experimental_strategy, sample_type, molecular_subtype)
# A tibble: 7 × 5
  Kids_First_Biospecimen_ID sample_id experimental_strategy sample_type molecular_subtype 
  <chr>                     <chr>     <chr>                 <chr>       <chr>             
1 BS_2PEY9Y3X               7316-7511 Targeted Sequencing   Tumor       LGG, KIAA1549-BRAF
2 BS_565NXM7G               7316-4439 Targeted Sequencing   Tumor       LGG, KIAA1549-BRAF
3 BS_FR42HY0Z               7316-7511 RNA-Seq               Tumor       LGG, wildtype     
4 BS_JN3S2H2Z               7316-4439 RNA-Seq               Tumor       LGG, wildtype     
5 BS_JZV7KSF4               7316-4439 WGS                   Tumor       LGG, wildtype     
6 BS_SSKCPAAX               7316-7511 WGS                   Normal      NA                
7 BS_WCX9HT5Y               7316-7511 WGS                   Tumor       LGG, wildtype 
```

The above samples belong to the same patient at different events. Clinical seq found the `KIAA1549::BRAF` fusion, however, in looking at all raw fusion caller output, the RNA-Seq samples do not contain the fusion and are therefore subtyped as WT. This needs investigation.

1. Are any of the RNA-Seq mismatched?
2. Are they low quality RNA-Seq?
3. Do we see evidence of the fusion in the CRAMs?
4. Are the RNA-Seq/WGS research samples completely mis-matched from the clinical seq?

**Comment by jharenza** [comment]
7316-4439 low quality rna-seq, removed from v13. Subtypes collapsed in v14.

**Comment by jharenza** [comment]
closed with #545
